src/coreComponents/constitutive/solid/PorousSolid.cpp

```diff
@@ -0,0 +1,66 @@
+/*
+ * ------------------------------------------------------------------------------------------------------------
+ * SPDX-License-Identifier: LGPL-2.1-only
+ *
+ * Copyright (c) 2018-2020 Lawrence Livermore National Security LLC
+ * Copyright (c) 2018-2020 The Board of Trustees of the Leland Stanford Junior University
+ * Copyright (c) 2018-2020 Total, S.A
+ * Copyright (c) 2019-     GEOSX Contributors
+ * All rights reserved
+ *
+ * See top level LICENSE, COPYRIGHT, CONTRIBUTORS, NOTICE, and ACKNOWLEDGEMENTS files for details.
+ * ------------------------------------------------------------------------------------------------------------
+ */
+
+
+/**
+ * @file PorousSolid.cpp
+ */
+
+#include "PorousSolid.hpp"
+#include "ElasticIsotropic.hpp"
+#include "ElasticTransverseIsotropic.hpp"
+#include "DruckerPrager.hpp"
+#include "DruckerPragerExtended.hpp"
+#include "Damage.hpp"
+#include "DamageSpectral.hpp"
+#include "DamageVolDev.hpp"
+#include "porosity/BiotPorosity.hpp"
+
+
+namespace geosx
+{
+
+using namespace dataRepository;
+
+namespace constitutive
+{
+
+template< typename SOLID_TYPE >
+PorousSolid< SOLID_TYPE >::PorousSolid( string const & name, Group * const parent ):
+  CoupledSolid< SOLID_TYPE, BiotPorosity >( name, parent )
+{}
+
+template< typename SOLID_TYPE >
+PorousSolid< SOLID_TYPE >::~PorousSolid()
+{}
```

Comment:
Same, could you use `default` impl?

Reply:
I have really just followed an existing pattern. It may very well be that it's not necessary though so if that's the case we should clean it up everywhere.